application/views/installer/precheck_view.php

```diff
@@ -13,6 +13,11 @@
         <?php $this->renderPartial('/installer/sidebar_view', compact('progressValue', 'classesForStep')); ?>
     </div>
     <div class="col-lg-9">
+        <?php if (empty($next) && empty($cookiesAllowed)): ?>
+            <div class="alert alert-warning" role="alert">
+                <?= gT("Cookies seem to be disabled. Please use the \"Check again\" button instead of refreshing the page."); ?>
```

Comment:
Cookie disabled or broken (bad server settings ?) 

Else, unsure on quote : why not real [Quotation mark](https://en.wikipedia.org/wiki/Quotation_mark)  ? `“Check again”`